frontend/compositions/map/map-helpers.tsx

```diff
@@ -0,0 +1,27 @@
+import * as d3 from "d3"
+
+
+
+
```

Comment:
Curious why we have so much leading space between imports & file content/exports in some of these files?

I don't have a particular preference for either (having it or not) but I do want us to be consistent & don't see that convention elsewhere - maybe a question for the team but I think we should either _have_ or _not have_ the leading line space in all files

Reply:
…because I have reading/writing difficulties and adding extra space everywhere helps a ton. I think we can all have our preferences with some clever prettier settings...I'll look into it!

Reply:
That's totally fair! I'm happy to have extra space if we use it consistently - prettier sounds like it could be a big help there - thanks for looking into it!